control-plane/grpc/proto/v1/volume/volume.proto

```diff
@@ -178,6 +178,8 @@ message PublishVolumeRequest {
   optional string target_node = 2;
   // share protocol
   optional VolumeShareProtocol share = 3;
+  /// Hosts allowed to access nexus.
+  repeated string frontend_hosts = 5;
 }
 
 // Republish a volume on a node by shutting down existing target
```

Comment:
Would we not the hosts field in republish as well? We do new nexus creation and sharing there as well, or is it going to be taken from older nexus?

Reply:
Yes. It is taken from old nexus during republish.